sdks/python/apache_beam/ml/transforms/handlers.py

```diff
@@ -447,22 +409,22 @@ def expand(
       raw_data_metadata = metadata_io.read_metadata(
           os.path.join(self.artifact_location, RAW_DATA_METADATA_DIR))
 
-    keyed_raw_data = (raw_data | beam.ParDo(_ComputeAndAttachUniqueID()))
+    keyed_raw_data = (raw_data)  #  | beam.ParDo(_ComputeAndAttachUniqueID()))
```

Comment:
leftover comment, also we no longer add  keys , so `keyed_` might not be the best name.

Reply:
Removed the keyed_ from variable names